app/src/main/kotlin/io/eugenethedev/taigamobile/ui/screens/main/MainActivity.kt

```diff
@@ -232,11 +235,22 @@ fun MainScreen(
         }
     }
 
+    val scope = rememberCoroutineScope()
+    val context = LocalContext.current
+    val onShowMessage: (Int) -> Unit = { message ->
```

Comment:
`showMessage` would look better

Reply:
Did you mean to change `onShowMessage` to `showMessage`?